packages/fxa-settings/src/components/Settings/ProductPromo/en.ftl

```diff
@@ -2,9 +2,12 @@
 
 product-promo-monitor =
  .alt = { -product-mozilla-monitor }
-product-promo-monitor-description = Find where your private info is exposed — and take it back
-product-promo-monitor-plus-description = Privacy Matters: Find where your private info is exposed and take it back
+product-promo-monitor-description-v2 = Find where your private info is exposed and take control
+# this message will only be shown to users eligible for a special promotion, based on their location (initially USA only)
+# $price - will be in the currency for the eligible location, etc., $8.25 for USA promo
+# /mo is 'per month'
+product-promo-monitor-special-promo-description = For { $price }/mo, save on { -product-mozilla-vpn-short }, { -product-mozilla-monitor-short }’s data-broker protection, and { -product-firefox-relay-short }’s unlimited email masks.
```

Comment:
@bcolsson Does Fluent support something like 
`price = { CURRENCY(amount: $number, currency: "USD") }` to better format the price?

Reply:
@vpomerleau - It's not possible to define the "currency" parameter natively in Fluent but you can format the variable using Intl.NumberFormat then pass it to the Fluent variable.
See: https://projectfluent.org/fluent/guide/functions.html#partially-formatted-variables

This also has the added benefit of not having to update the string if/when we want to use it for another currency. (Assuming we can dynamically apply the correct currency in whatever function we wrap the variable in for formatting.)